templates/coderd.yaml

```diff
@@ -194,7 +194,7 @@ spec:
               value: "/etc/ssl/certs/client/tls.key"
             {{- end }}
             - name: CODER_TRUSTED_HOSTNAMES
```

Comment:
should this be renamed as well?

Reply:
It doesn't need to be since it's not user facing, but if you want me to rename it I can

Reply:
I think that's preferable to avoid confusion if it's not too much work to change it, but no big deal